### PR TITLE
Markdown  file added

### LINKED ISSUE #7: new lesson: ArrayList In Java
### ⭐ Request Type

Add details documentation/content for the lesson `ArrayList In Java`.
you can start your contribution from [here](https://github.com/Utkarsh1504/DSA-Java/blob/main/lessons/arraylist.md)

### 💻 Location

This lesson is present under the
Section: `4`
Lesson: `B`
Location Order: `4B`
File-Name: `arraylist.md`

### ❌ Additional Info

You can take the help of other markdown files in the /lesson [directory](https://github.com/Utkarsh1504/DSA-Java/blob/main/lessons/operators.md)

This lesson must consist of at least the following topic:
1. Introduction
2. Syntax+Internal Working
3. code examples based on arraylist

### PR DESCRIPTION
**Pull request by preeti-14-7**
Fixes #7 

#### What Changes proposed in this pull request:

 - I have added the detail explanation of ArrayList in java.
- I have added some code snippet also.

#### Checklist

- [X] Added description of change
- [X] Added file name matches [File name guidelines](https://github.com/Utkarsh1504/DSA-Java/blob/main/CONTRIBUTING.md#requesting-a-lesson)
- [X] Added code examples, test must pass
- [X] Added documentation so that the program is self-explanatory and educational.
- [X] PR title follows semantic [commit guidelines](https://github.com/Utkarsh1504/DSA-Java/blob/main/CONTRIBUTING.md#requesting-a-lesson)
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: 
This is a documentation of DSA in Java

### CODE REVIEW

**Review by Utkarsh1504 — CHANGES_REQUESTED** [review]
Just Some small tweaks and your PR is ready for merge.
The content looks really great!

File: lessons/arraylist.md [hunk 1]
```diff
@@ -5,3 +5,184 @@ order: "4B"
 section: "Arrays"
 description: "learn arrays"
 ---
+ ## ArrayList in java 
+ <hr>
```
Comment (Utkarsh1504) [inline lessons/arraylist.md:9]:
```suggestion

```
remove these two lines, lesson heading will be there on the site by default

File: lessons/arraylist.md [hunk 2]
```diff
@@ -5,3 +5,184 @@ order: "4B"
...
+
+## Basic Operation in Java ArrayList :
+The ArrayList class have some methods to perform different operations on arraylists. Let's see some of most used methods.
+
+- Add elements
+- Access elements
+- Change elements
+- Remove elements
+
+## Add Elements :
```
Comment (Utkarsh1504) [inline lessons/arraylist.md:70]:
```suggestion
### Add Elements: 
```
make the heading `h3`

File: lessons/arraylist.md [hunk 3]
```diff
@@ -5,3 +5,184 @@ order: "4B"
...
+    // add() method 
+    playerName.add("Virat");
+    playerName.add("Dhoni");
+    playerName.add("Sachin");
+	//print 
+    System.out.println("ArrayList: " + playerName);
+  }
+}
+```
+## Output
```
Comment (Utkarsh1504) [inline lessons/arraylist.md:89]:
```suggestion
### Output:
```

File: lessons/arraylist.md [hunk 4]
```diff
@@ -5,3 +5,184 @@ order: "4B"
...
+    System.out.println("ArrayList: " + playerName);
+
+	// get() method
+    String name1 = playerName.get(1);
+    System.out.println(name1);
+  }
+}
+
+```
+## Output
```
Comment (Utkarsh1504) [inline lessons/arraylist.md:117]:
```suggestion
### Output:
```

File: lessons/arraylist.md [hunk 5]
```diff
@@ -5,3 +5,184 @@ order: "4B"
...
+    System.out.println("ArrayList: " + playerName);
+
+
+    // change the element of the array list
+    languages.set(2, "Rohit");
+    System.out.println("New ArrayList: " + playerName);
+  }
+}
+```
+## Output
```
Comment (Utkarsh1504) [inline lessons/arraylist.md:144]:
```suggestion
### Output:
```

File: lessons/arraylist.md [hunk 6]
```diff
@@ -5,3 +5,184 @@ order: "4B"
...
+    // remove element 
+    String name1 = playerName.remove(2);
+	//print list after removal
+    System.out.println("New ArrayList: " + playerName);
+	//removed elements
+    System.out.println("Removed: " + name1);
+  }
+}
+```
+## Output
```
Comment (Utkarsh1504) [inline lessons/arraylist.md:175]:
```suggestion
### Output:
```

File: lessons/arraylist.md [hunk 7]
```diff
@@ -5,3 +5,184 @@ order: "4B"
...
+
+```java
+import java.util.ArrayList;
+
+class Main {
+  public static void main(String[] args) {
+    ArrayList<String> languages = new ArrayList<>();
+
+    // add() method without the index parameter
+    playerName.add("Virat");
```
Comment (Utkarsh1504) [inline lessons/arraylist.md:132]: playerName is not defined??

File: lessons/arraylist.md [hunk 8]
```diff
@@ -5,3 +5,184 @@ order: "4B"
...
+
+```
+- remove() : To remove the elements from the list
+
+```java
+import java.util.ArrayList;
+
+class Main {
+  public static void main(String[] args) {
+    ArrayList<String> animals = new ArrayList<>();
```
Comment (Utkarsh1504) [inline lessons/arraylist.md:157]: define playerName instead of animals

**Review by Utkarsh1504 — CHANGES_REQUESTED** [review]
please do these changes

**Comment by preeti-14-7** [comment]
Check it again.

**Review by Utkarsh1504 — APPROVED** [review]

**Comment by Utkarsh1504** [comment]
Thank you for your work!🎉🎉🎈

**Pull request merged**